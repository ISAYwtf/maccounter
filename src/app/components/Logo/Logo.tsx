import React from 'react'
// import styles from './Logo.module.css'

const Logo: React.FC = () => (
    <div className="txt--logo">
        <span className="txt--orange">m.</span>
        <span>accounter</span>
    </div>
)

export default Logo
