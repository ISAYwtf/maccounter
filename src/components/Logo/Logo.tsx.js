import React from "react"
import styles from "./Logo.module.css"

const Logo = () =>
    <div className={"txt--logo"}>
        <span className={"txt--orange"}>m.</span>
        <span>accounter</span>
    </div>

export default Logo
