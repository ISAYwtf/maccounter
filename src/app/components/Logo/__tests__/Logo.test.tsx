import React from 'react'
import { render } from '@testing-library/react'
import Logo from '../Logo'

describe('Logo', () => {
    const renderComponent = <Logo />

    it('render Logo', () => {
        const { asFragment } = render(renderComponent)
        expect(asFragment()).toMatchSnapshot()
    })
})
