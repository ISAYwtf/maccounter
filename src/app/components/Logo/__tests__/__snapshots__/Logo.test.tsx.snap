// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo render Logo 1`] = `
<DocumentFragment>
  <div
    class="txt--logo"
  >
    <span
      class="txt--orange"
    >
      m.
    </span>
    <span>
      accounter
    </span>
  </div>
</DocumentFragment>
`;
